### PR TITLE
Phase 5: Wider/Deeper Transolver — Model Capacity Exploration (8 parallel)

### PR DESCRIPTION
**Pull request by tcapelle**
## Hypothesis

The current Transolver uses n_hidden=192, n_layers=3 with 96 slices (~37M params, 36.2 GB). With 96 GB VRAM available, we have significant headroom. A wider model (256 or 384 hidden) or deeper model (4-5 layers) could capture more complex flow patterns, especially for the high-variance p_in metric.

**Why this should work:**
- The slice count sweep (askeladd #1914) showed that 96 slices is optimal and more slices hurts throughput. But that explored attention breadth, not model depth/width.
- Wider models have more capacity per layer without the vanishing gradient risk of deeper models.
- 96 GB VRAM budget is only ~38% utilized — significant room for a wider model.
- Wider models with the same number of layers compile efficiently.
- The key risk is that wider models may need more epochs to converge, and we're already at ~159 epochs in 180 min.

**What to watch for:** If wider models get significantly fewer epochs (like 192/256 slices did), the throughput penalty may cancel the capacity benefit.

## Instructions

See first comment for implementation details.

## Baseline (8-seed)
val/loss 0.404+/-0.004, p_in 13.33+/-0.58, p_oodc 8.37+/-0.22, p_tan 33.57+/-0.44, p_re 24.58+/-0.13

### CODE REVIEW

**Comment by tcapelle** [comment]
## Detailed Implementation

No code changes needed — just use different `--n_hidden` and `--n_layers` flags. The model already supports these.

### GPU Sweep (8 parallel)

| GPU | Config | wandb_name | Expected VRAM |
|-----|--------|------------|---------------|
| 0 | `--n_hidden 256 --n_layers 3` | askeladd/w256-l3 | ~55 GB |
| 1 | `--n_hidden 256 --n_layers 4` | askeladd/w256-l4 | ~65 GB |
| 2 | `--n_hidden 384 --n_layers 3` | askeladd/w384-l3 | ~75 GB |
| 3 | `--n_hidden 192 --n_layers 4` | askeladd/l4 | ~45 GB |
| 4 | `--n_hidden 192 --n_layers 5` | askeladd/l5 | ~55 GB |
| 5 | `--n_hidden 256 --n_layers 3 --lr 1.5e-4` | askeladd/w256-l3-lrlower | ~55 GB |
| 6 | Baseline (192, 3 layers) seed 42 | askeladd/baseline-s42 | 36 GB |
| 7 | Baseline seed 43 | askeladd/baseline-s43 | 36 GB |

All use `--wandb_group phase5/wider-deeper`.

Base command:
```bash
python train.py --agent askeladd --wandb_name "askeladd/<variant>" \
  --wandb_group phase5/wider-deeper \
  --field_decoder --adaln_output --use_lion --lr 2e-4 \
  --aug aoa_perturb --aug_full_dsdf_rot --high_p_clamp \
  --n_layers 3 --slice_num 96 --tandem_ramp \
  --domain_layernorm --domain_velhead --ema_decay 0.999 \
  --weight_decay 5e-5 --cosine_T_max 180 --disable_pcgrad \
  --pressure_first --pressure_deep \
  --n_hidden <H> --n_layers <L>
```

### Notes
- Monitor VRAM usage and epoch time — the key question is whether wider/deeper models get enough epochs to converge.
- If w384-l3 OOMs, try with `--batch_size 3` instead.
- The w256-l3 with lower lr (1.5e-4) tests whether wider models need a smaller learning rate.
- Log epochs completed and time per epoch to compare throughput.
